src/OrganisationRegistry/Organisation/Organisation.cs

```diff
@@ -1530,12 +1530,15 @@ public void UpdateCurrentOrganisationParent(DateTime today)
             var events = new List<IEvent>();
 
             if (_currentOrganisationParent != null && !_currentOrganisationParent.Validity.OverlapsWith(today))
+            {
```

Comment:
Subjective, but not a fan of these

Reply:
was een restant van debugging stuff ;) .. gone.